docs/linux-instructions.md

```diff
@@ -88,6 +88,10 @@ shortcuts.
  3. You can launch the application by executing `dotnet run` in the `src/WebUI` directory.
 
  4. You can run the tests by executing `dotnet test` in the root directory.
+ 
+ ## Setting up the auth provider
+ 
+ You'll also need to set up the auth provider. See [this instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.
```

Comment:
I am not a native speaker, is "those instructions" correct?
```suggestion
You will also need to set up the authentication provider. See [those instructions](https://github.com/codidact/authentication/blob/develop/docs/development.md) for more information.
```

Reply:
Isnt it these instructions?

Reply:
I just read an article about those *versus* these and I still do not know.

Reply:
I am not a native speaker either, but "these" sounds more correct IMO. I'm going to make this change and then merge.